### PR TITLE
Add fauci and vaccine message react

### PR DESCRIPTION
**Pull request by RadarRadarRadar**

### CODE REVIEW

File: src/text-channel-controllers/auto-react.controller.ts [hunk 1]
```diff
@@ -42,4 +42,11 @@ export class AutoReactController {
         const gundamEmoji = message.guild.emojis.cache.find(e => e.name === 'gundam');
         await message.react(gundamEmoji);
     }
+
+    @DiscordEvent('message')
+    @UseGuards(ContainsText('fauci|vaccine'))
```
Comment (haroldwaters) [inline src/text-channel-controllers/auto-react.controller.ts:47]: delete fauci, but i'll take vaccine

File: src/text-channel-controllers/auto-react.controller.ts [hunk 2]
```diff
@@ -42,4 +42,11 @@ export class AutoReactController {
...
+    public async warrenSnake(@Ctx() ctx: DiscordContext) {
+        const message = ctx.getArgByIndex(0);
+        await message.react('💉');
```
Comment (haroldwaters) [inline src/text-channel-controllers/auto-react.controller.ts:50]: maybe add a troll emoji in there too? maybe 5️🇬?

File: src/text-channel-controllers/post-tracking.controller.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+box
```
Comment (haroldwaters) [inline src/text-channel-controllers/post-tracking.controller.ts:1]: delete me

File: src/text-channel-controllers/auto-react.controller.ts [hunk 4]
```diff
@@ -42,4 +42,11 @@ export class AutoReactController {
         const gundamEmoji = message.guild.emojis.cache.find(e => e.name === 'gundam');
         await message.react(gundamEmoji);
     }
+
+    @DiscordEvent('message')
+    @UseGuards(ContainsText('fauci|vaccine'))
+    public async warrenSnake(@Ctx() ctx: DiscordContext) {
```
Comment (haroldwaters) [inline src/text-channel-controllers/auto-react.controller.ts:48]: change the method name

File: src/text-channel-controllers/auto-react.controller.ts (hunk 2 above)
Comment (RadarRadarRadar) [reply src/text-channel-controllers/auto-react.controller.ts:50]: I like 5G

Comment (haroldwaters) [reply src/text-channel-controllers/auto-react.controller.ts:50]: perhaps have a coin flip to determine too, sometimes 5g, sometimes vaccine

Comment (RadarRadarRadar) [reply src/text-channel-controllers/auto-react.controller.ts:50]: Actually went with a sheep instead

Comment (haroldwaters) [reply src/text-channel-controllers/auto-react.controller.ts:50]: love it

**Review by RadarRadarRadar — COMMENTED** [review]
Requested changes made.

File: src/text-channel-controllers/auto-react.controller.ts [hunk 5]
```diff
@@ -42,4 +42,13 @@ export class AutoReactController {
         const gundamEmoji = message.guild.emojis.cache.find(e => e.name === 'gundam');
         await message.react(gundamEmoji);
     }
+
+    @DiscordEvent('message')
+    @UseGuards(ContainsText('vaccine'))
+    public async vaccineSheep(@Ctx() ctx: DiscordContext) {
+        const message = ctx.getArgByIndex(0);
+        await message.react('💉');
+        await message.react('🐑');
```
Comment (haroldwaters) [inline src/text-channel-controllers/auto-react.controller.ts:51]: Do a coin flip to determine which one goes, don't do both.

File: src/text-channel-controllers/auto-react.controller.ts [hunk 6]
```diff
@@ -42,4 +42,13 @@ export class AutoReactController {
...
+        await message.react('💉');
```
Comment (haroldwaters) [inline src/text-channel-controllers/auto-react.controller.ts:52]: You've got this react twice

**Review by haroldwaters — CHANGES_REQUESTED** [review]

File: src/text-channel-controllers/auto-react.controller.ts (hunk 5 above)
Comment (RadarRadarRadar) [reply src/text-channel-controllers/auto-react.controller.ts:51]: Sure, I was modeling it off of the rat Pete method but coin flip is funny.

File: src/text-channel-controllers/auto-react.controller.ts (hunk 6 above)
Comment (RadarRadarRadar) [reply src/text-channel-controllers/auto-react.controller.ts:52]: Ah I forget that the emojis double up and don't show one after another like in a text.

**Review by RadarRadarRadar — COMMENTED** [review]
Flip that coin!

**Review by haroldwaters — APPROVED** [review]

**Pull request merged**